### PR TITLE
wait logic for PDNS GLB Destroy

### PR DESCRIPTION
**Pull request by nimisha0092**

### CODE REVIEW

File: ibm/resource_ibm_private_dns_glb.go [hunk 1]
```diff
@@ -327,3 +334,34 @@ func suppressPDNSGlbNameDiff(k, old, new string, d *schema.ResourceData) bool {
 	}
 	return false
 }
+
+func isWaitForLoadBalancerDeleted(LoadBalancer *dnssvcsv1.DnsSvcsV1, d *schema.ResourceData, timeout time.Duration) (interface{}, error) {
+	idset := strings.Split(d.Id(), "/")
+	log.Printf("Waiting for PDNS GLB (%s) to be deleted.", idset[2])
+	stateConf := &resource.StateChangeConf{
+		Pending:    []string{"retry", pdnsGLBDeleting},
```
Comment (KumarGanesanIBM) [inline ibm/resource_ibm_private_dns_glb.go:342]: remove `"retry",` from this line

File: ibm/resource_ibm_private_dns_glb.go [hunk 2]
```diff
@@ -327,3 +334,34 @@ func suppressPDNSGlbNameDiff(k, old, new string, d *schema.ResourceData) bool {
...
+		Target:     []string{pdnsGLBDeleted, ""},
```
Comment (KumarGanesanIBM) [inline ibm/resource_ibm_private_dns_glb.go:343]: remove `,""` from this line.

File: ibm/resource_ibm_private_dns_glb.go [hunk 3]
```diff
@@ -327,3 +334,34 @@ func suppressPDNSGlbNameDiff(k, old, new string, d *schema.ResourceData) bool {
...
+	}
+
+	return stateConf.WaitForState()
+}
+
+func isVLoadBalancerDeleteRefreshFunc(LoadBalancer *dnssvcsv1.DnsSvcsV1, d *schema.ResourceData) resource.StateRefreshFunc {
+	return func() (interface{}, string, error) {
+		idset := strings.Split(d.Id(), "/")
+		deletelbOptions := LoadBalancer.NewDeleteLoadBalancerOptions(idset[0], idset[1], idset[2])
+		response, err := LoadBalancer.DeleteLoadBalancer(deletelbOptions)
```
Comment (KumarGanesanIBM) [inline ibm/resource_ibm_private_dns_glb.go:357]: you should call `GetLoadBalancer` method here.

**Review by KumarGanesanIBM — CHANGES_REQUESTED** [review]

File: ibm/resource_ibm_private_dns_glb.go (hunk 2 above)
Comment (nimisha0092) [reply ibm/resource_ibm_private_dns_glb.go]: handled

File: ibm/resource_ibm_private_dns_glb.go (hunk 3 above)
Comment (nimisha0092) [reply ibm/resource_ibm_private_dns_glb.go]: Handled

**Review by KumarGanesanIBM — APPROVED** [review]

**Comment by KumarGanesanIBM** [comment]
@kavya498 can you please review this once?

**Comment by KumarGanesanIBM** [comment]
@kavya498 
Could yo please let me know why this is still not reviewed?

Thanks,
KumarG.

**Review by kavya498 — APPROVED** [review]

**Comment by kavya498** [comment]
Review is done.. @hkantare ll merge it soon..
Thanks.

**Pull request merged**